### DISCUSSION TITLE
How can I stop storybook from grouping attributes, properties and story args?

### DISCUSSION THREAD

**Discussion by fdendorfer** [storybookjs/storybook#25237] [Help]
### Summary

First up, I'm using WebComponents with a custom-elements.json manifest, which causes them to be grouped in the first place. In the reproduction I created a component demo-button, which has a property apiTarget, an private state property _timesClicked and a story arg width, but in the resulting storybook they're grouped by autodocs like the screenshot below. How can I explicitly stop the grouping of these properties or at least rename the groups or filter story args and private fields out? The private fields I found I can filter out via a custom plugin in custom-elements-manifest.config.mjs, but the width property is always there unwantingly.

![image](https://github.com/storybookjs/storybook/assets/23694947/122086a9-ca57-4197-a8d3-f8f04bcff546)

### Create a reproduction

https://stackblitz.com/edit/github-7rvkco

**Comment by chantastic** [comment]
These are all of the currently supported options on the ArgsTable component: https://github.com/storybookjs/storybook/blob/next/code/addons/docs/docs/props-tables.md#customization

It’s possible that you may need to customize, using DocsPage.

[Storybook delegates to libraries in the UI Library ecosystem](https://github.com/storybookjs/storybook/blob/next/code/addons/docs/docs/props-tables.md#known-limitations).
Further details on that implementation are available here: https://github.com/storybookjs/storybook/blob/next/code/addons/docs/web-components/README.md#props-tables

**Reply by fdendorfer** [reply]
Okay, so I can exclude them with the following snippet, but I still cannot stop the property "apiTarget" from being grouped under "ATTRIBUTES" and the ts compiler will always complain, that "_timesClicked" does not belong on meta, even if specified it's type as Meta<IDemoButtonStory | DemoButton> and will only stop at <any>. Is that correct and is this the best solution?

```typescript
argTypes: {
    _timesClicked: {
      if: { arg: 'false' },
    },
    apiTarget: {
      name: 'API Target',
    },
    content: {
      if: { arg: 'false' },
    },
}
```

**Comment by dj-fiorex** [comment]
Is there a way to modify the input to the table? to create for example more table than one (i have a lot of properties) 
another way could be to have an array of subcategories instead of a single one
Please if there is a way to override this let me know!
Thanks!
